### PR TITLE
INFENG-5814 add struct name to required field error

### PR DESCRIPTION
**Pull request by blakeroberts-wk**
https://jira.atl.workiva.net/browse/INFENG-5814

@Workiva/messaging-pp @Workiva/product2

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on HipChat: InfoSec Forum._

File: compiler/generator/golang/generator.go [hunk 1]
```diff
@@ -638,7 +638,7 @@ func (g *Generator) generateRead(s *parser.Struct, sName string) string {
 		if field.Modifier == parser.Required {
 			fName := snakeToCamel(field.Name)
 			contents += fmt.Sprintf("\tif !isset%s {\n", fName)
-			errorMessage := fmt.Sprintf("Required field %s is not set", fName)
+			errorMessage := fmt.Sprintf("Required field '%s' is not present in struct %s", s.Name, fName)
```
Comment (brianshannan-wf) [inline compiler/generator/golang/generator.go]: `s.Name` and `fName` are backward

File: compiler/generator/python/generator.go [hunk 2]
```diff
@@ -574,7 +574,7 @@ func (g *Generator) generateValidate(s *parser.Struct) string {
 		for _, field := range s.Fields {
 			if field.Modifier == parser.Required {
 				contents += fmt.Sprintf(tabtab+"if self.%s is None:\n", field.Name)
-				contents += fmt.Sprintf(tabtabtab+"raise TProtocol.TProtocolException(type=TProtocol.TProtocolException.INVALID_DATA, message='Required field %s is unset!')\n", field.Name)
+				contents += fmt.Sprintf(tabtabtab+"raise TProtocol.TProtocolException(type=TProtocol.TProtocolException.INVALID_DATA, message='Required field '%s' is not present in struct %s')\n", s.Name, field.Name)
```
Comment (brianshannan-wf) [inline compiler/generator/python/generator.go]: `s.Name` and `fName` are backward

File: compiler/generator/dartlang/generator.go [hunk 3]
```diff
@@ -835,7 +835,7 @@ func (g *Generator) generateRead(s *parser.Struct) string {
 		if field.Modifier == parser.Required && g.isDartPrimitive(field.Type) {
 			fName := toFieldName(field.Name)
 			contents += fmt.Sprintf(tabtab+"if(!__isset_%s) {\n", fName)
-			contents += fmt.Sprintf(tabtabtab+"throw new thrift.TProtocolError(thrift.TProtocolErrorType.UNKNOWN, \"Required field '%s' was not present in struct %s\");\n", fName, s.Name)
```
Comment (brianshannan-wf) [inline compiler/generator/dartlang/generator.go]: Pretty nitpicky, but if we're putting the field name in quotes we should probably match that behaviour in the struct name

Comment (blakeroberts-wk) [reply compiler/generator/dartlang/generator.go]: Fair enough.

**Comment by danielnovotny-wf** [comment]
+1

**Pull request merged**